### PR TITLE
Update Home Page tile height

### PR DESCRIPTION
**Pull request by jasonTelanoff**
Also Changed Donate Now button text to read differently and stand out more.

**Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).**
- [ ] REQUIRED: Do you have an Issue **assigned to you** within the v1 milestone for this PR?  Put the Issue number here:
- [ ] Provided detailed pull request description and a succinct title (consider template below for guidance).
- [ ] Tested your changes, especially after any code review iterations.
- [ ] Included any relevant screenshots of UI updates.
- [ ] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [ ] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [ ] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

After all boxes above are checked, request and receive an Approved review from any team member knowledgable in the area (TODO team member list).  Once approved, the team member will assign your review to a Committer or use the `needs-merge` label.

## What does this PR accomplish?

improves look & feel, easier access to donation button

decrease interior padding of buttons
move "how to help" text into donate button and increased size and weight.

#697

## Did you add any dependencies?
no

## How did you test the change?
Tested on physical Galaxy S10

Screenshot_20200330-195455

### CODE REVIEW

File: client/flutter/lib/pages/home_page.dart [hunk 1]
```diff
@@ -147,19 +148,19 @@ class _HomePageState extends State<HomePage> {
           ),
           SliverList(
             delegate: SliverChildListDelegate.fixed([
+            //   Padding(
```
Comment (hspinks) [inline client/flutter/lib/pages/home_page.dart]: remove commented code

File: client/flutter/lib/pages/home_page.dart [hunk 2]
```diff
@@ -168,7 +169,8 @@ class _HomePageState extends State<HomePage> {
                     child: Row(
                       mainAxisAlignment: MainAxisAlignment.spaceBetween,
                       children: <Widget>[
-                        Text(S.of(context).homePagePageSliverListDonate),
+                        // Text(S.of(context).homePagePageSliverListDonate),
```
Comment (hspinks) [inline client/flutter/lib/pages/home_page.dart]: let's please stick to the design spec on this

**Comment by theswerd** [comment]
@advayDev1 changes have been addressed

**Comment by hspinks** [comment]
@jasonTelanoff @theswerd can you please resolve conflicts on this so we can merge?

**Comment by advayDev1** [comment]
Will merge once new preview is up

**Comment by advayDev1** [comment]
iPhone X seems to cutoff
Screen Shot 2020-03-31 at 9 40 08 PM

File: client/flutter/lib/pages/home_page.dart [hunk 3]
```diff
@@ -47,6 +47,7 @@ class _HomePageState extends State<HomePage> {
 
   @override
   Widget build(BuildContext context) {
+    double tileHeightFactor = 0.7;
```
Comment (advayDev1) [inline client/flutter/lib/pages/home_page.dart:50]:
```suggestion
    double tileHeightFactor = 0.73;
```

**Pull request merged**